docs/user_guide/reporting/Supervisor Notifications/Non Alerted Document Report

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Notifications
```

Comment:
Seems weird to have a page called Notifications under a page called Supervisor Notifications.

Reply:
The other notification tab will be retired once this is released